docs/components/password/password-demo.js

```diff
@@ -0,0 +1,95 @@
+            hasAsterisk: false,
+            hasOptional: false,
+            hasError : false,
+            hasHelpText : false,
+            isDisabled: false,
+            isRequired: false,
+            label: 'Pas$w0rd',
+            errorTextToDisplay: false,
+            helpTextToDisplay: false,
+            displayTrue :true,
+            displayFalse: false,
+            helpText: 'At least 8 characters with 1 uppercase, 1 lowercase, and 1 number.',
+            errorText: 'Please Enter Valid Password',
+
+        },
+        computed: {
+            attrDisabled: function () {
+                return (this.isDisabled ? 'disabled' : '');
+            },
+            attrRequired: function () {
+                return (this.isRequired ? 'required' : '');
+            },
+            lblClasses: function () {
+                let classes = [];
+
+                if (this.hasAsterisk) {
+                    classes.push('hxRequired');
+                }
+
+                if (this.hasOptional) {
+                    classes.push('hxOptional');
+                }
+
+                let classNames = classes.join(' ');
+
+                return (classNames === '' ? '' : `class="${classNames}"`);
+            },
+            helpTextClass: function () {
+                let textClasses = [];
+
+                if (this.hasHelpText) {
+                    textClasses.push('hxHelpText');
+                    this.helpTextToDisplay = this.displayTrue;
+                } else {
+                    this.helpTextToDisplay = this.displayFalse;
+                }
+
+                let textClassNames = textClasses.join(' ');
+
+                return (textClassNames === '' ? '' : `class="${textClassNames}"`);
+            },
+            errorTextClass: function () {
+                let errorTextClasses = [];
+
+                if (this.hasError) {
+                    errorTextClasses.push('hxErrorText');
+                    this.errorTextToDisplay = this.displayTrue;
+                } else {
+                    this.errorTextToDisplay = this.displayFalse;
+                }
+
+                let errorTextClassNames = errorTextClasses.join(' ');
+
+                return (errorTextClassNames === '' ? '' : `class="${errorTextClassNames}"`);
+            },
+
+            snippet: function () {
+                return Util.snippet(`
+                    <hx-password-control>
+                      <input
+                        id="pwdDemo"
+                        type="password"
+                        ${this.attrDisabled}
+                        ${this.attrRequired}
+                      />
+                      <label
+                        for="pwdDemo"
+                        ${this.lblClasses}
+                      >
+                         Password
+                      </label>
+                      <p ${this.helpTextClass}> ${this.helpText} </p>
+                      <p ${this.errorTextClass}> ${this.errorText} </p>
```

Comment:
1. Please update the logic to toggle on `hidden` attribute.   Keep the `hxErrorText` class.  You will add/remove `hidden`.
1. It will look something like this:

```js
<p class="hxErrorText" ${this.isErrorTextHidden}> ${this.helpText} </p>
```

Reply:
Changes have been done, please review.